pontoon/base/templates/translate.html

```diff
@@ -304,7 +304,8 @@ <h2></h2>
         </ul>
       </nav>
 
-      <textarea {% if not user.is_authenticated() %} readonly {% endif %} id="translation" placeholder="Type translation and press Enter to save"></textarea>
+      <textarea {% if user.is_authenticated() %} id="translation" placeholder="Type translation and press Enter to save" 
+      {% else %} id="translation-no-login"  placeholder="Sign in below to begin translating."  {% endif %}></textarea>
```

Comment:
Please make sure that you keep the `id` unchanged and that you don't remove the `readonly attribute`.

In CSS you'd then reference editor for non-authenticated users with `#editor textarea[readonly] {`, which will also allow you to only change the background color (and remove `(min-)height`).

Note that there's another `textarea` below (line 314), where you also need to change the placeholder text based on `is_authenticated` status.

Reply:
Thank you for the corrections! I've made and committed your suggested changes.